src/one_dragon/base/cv_process/cv_service.py

```
@@ -9,23 +10,61 @@
+
+
+def reset_current_plugin(token: contextvars.Token) -> None:
+    """恢复插件上下文"""
+    _current_plugin.reset(token)
+
 
 class CvService:
     """
     一个纯净的、无UI依赖的CV流水线服务
     负责流水线的加载、保存、执行等核心功能
+    流水线来源为多目录：主仓 + 各插件（命名空间 插件名::流水线名）
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)